meta-oe/recipes-connectivity/linuxptp/linuxptp_3.1.bb

```diff
@@ -10,14 +10,12 @@ SRC_URI = "http://sourceforge.net/projects/linuxptp/files/v${PV}/linuxptp-${PV}.
 SRC_URI[md5sum] = "2264cb69c9af947028835c12c89a7572"
 SRC_URI[sha256sum] = "f58f5b11cf14dc7c4f7c9efdfb27190e43d02cf20c3525f6639edac10528ce7d"
 
-EXTRA_OEMAKE = "ARCH=${TARGET_ARCH} EXTRA_CFLAGS='${CFLAGS}'"
+EXTRA_OEMAKE = "ARCH=${TARGET_ARCH} EXTRA_CFLAGS='${CFLAGS}' mandir=${mandir}"
```

Comment:
should this `inherit manpages` perhaps ?

Reply:
The man pages in linuxptp is not built, they are in the repo from the beginning. There is neither a configuration option to enable/disable the installation of them. (https://salsa.debian.org/multimedia-team/linuxptp/-/blob/master/makefile#L86) For that reason I don't see any need for 'inherit manpages' that create the PACKAGECONFIG manpages.

Then I see in manpages.bbclass that there is some indexing in pkg_postinst (http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/classes/manpages.bbclass#n15) that probably coudl be useful for linuxptp.

So I'm a bit unsure, please advise if you have an opinion. We can probably make the assumption that inherit manpages does not harm anything.